docs/content/en/docs/quickstart/_index.md

```diff
@@ -178,6 +170,39 @@ is completed, you should see the changes reflected in the output in the terminal
 If you prefer building and deploying once at a time, run `skaffold run`.
 Skaffold will perform the workflow described in `skaffold.yaml` exactly once.
 
+{{% /tab %}}
+
+{{% tab "CLOUD CODE" %}}
+
+Follow these quickstart guides if you're using Skaffold with the [Cloud Code]({{< relref "../install/#managed-ide" >}}) IDE extensions:
+
+### [Cloud Code for VSCode](https://cloud.google.com/code/docs/vscode/quickstart-k8s)
+
+Create, locally develop, debug, and run a Kubernetes application with Cloud Code for VSCode.
+
+![vscode](/images/cloud-code-quick-deploy.gif)
```

Comment:
Can we make these gifs also link to their respective quickstart guide? They take up a lot of screen space so I think it might be nice for them to link to something

Reply:
done.